cypress/page/payment-step.page.ts

```diff
@@ -1,20 +1,26 @@
 class PaymentStepPage {
-    private bankWireClick: string;
+    private bankWire: string;
     private proceedToPayClick: string;
     private messagePaymentComplete: string;
 
     constructor() {
-        this.bankWireClick = ".bankwire";
+        this.bankWire = ".bankwire";
         this.proceedToPayClick = "#cart_navigation > .button > span";
         this.messagePaymentComplete = "#center_column > div > p > strong";
 
     }
 
-    public paymentSuccessfully(): void {
-            cy.get(this.bankWireClick).click();
+    public selectBankWire(): void {
+            cy.get(this.bankWire).click();              
+    }
+
+    public confirmOrder(): void {
             cy.get(this.proceedToPayClick).click();
+    }
+
+    public getConfirmationMessage(): void {
```

Comment:
you can change the return type to any, then make the method to return the title, and move the should part to the test file
`return cy.get(this.messagePaymentComplete)`